packages/@webex/plugin-cc/test/unit/spec/cc.ts

```diff
@@ -714,5 +719,38 @@ describe('webex.cc', () => {
         {module: CC_FILE, method: 'silentRelogin'}
       );
     });
+  
+    it('should handle errors during silent relogin', async () => {
+      const error = new Error('Error while performing silentReLogin');
+      jest.spyOn(webex.cc.services.agent, 'reload').mockRejectedValue(error);
+  
+      await expect(webex.cc['silentRelogin']()).rejects.toThrow(error);
+    });
+  
+    it('should update agentConfig with deviceType during silent relogin', async () => {
```

Comment:
Please check the coverage report for the coverage of switch cases that has been added

Reply:
Yup, I've added them as part of the tests and as a new one below this. We test all the switch cases (including the error).

Reply:
@Kesari3008 please resolve this comment